src/Microsoft.ML.Transforms/CustomMappingCatalog.cs

```diff
@@ -46,5 +46,64 @@ public static class CustomMappingCatalog
             where TSrc : class, new()
             where TDst : class, new()
             => new CustomMappingEstimator<TSrc, TDst>(catalog.GetEnvironment(), mapAction, contractName, inputSchemaDefinition, outputSchemaDefinition);
+
+        /// <summary>
+        /// Create a <see cref="StatefulCustomMappingEstimator{TSrc, TState, TDst}"/>, which applies a custom mapping of input columns to output columns,
+        /// while allowing a per-cursor state.
```

Comment:
Are we adding this more for TLC parity or is there a customer ask for this? (Not opposed to the former, just curious)

Reply:
This is for parity, I don't know about customer asks.

---
In reply to: [399043928](https://github.com/dotnet/machinelearning/pull/4676#discussion_r399043928) [](ancestors = 399043928)